src/machine/machined.c

```diff
@@ -70,6 +70,11 @@ void manager_free(Manager *m) {
 
         assert(m);
 
+        while (m->operations)
+                operation_free(m->operations);
```

Comment:
hmm, so usually we use the (void) casting only for cases where the returned value indicates an error, that one should normally not ignore. But operation_free() returns NULL always, and no error condition. The only reason it returns anything at all is to allow `o = operation_unref(o);`... Also see CODING_STYLE.